docs/operate/get-started/supported-hardware/_index.md

```diff
@@ -117,11 +119,11 @@ After [installing `viam-server` or `viam-micro-server`](/operate/get-started/set
 
 In addition to physical hardware, there are "virtual" hardware modules that do not directly drive any physical hardware, but rather augment physical hardware with another layer of abstraction, or add other functionality, for example:
 
-- [A "camera" that takes a camera feed from a physical camera, and crops it, overlays it, or otherwise transforms the output](/operate/reference/components/camera/transform/)
```

Comment:
Curious if this is being intentionally removed?

Reply:
This was Rand feedback--that this particular module is wonky (hogs CPU) so maybe not the best example to showcase. I realize I'm replacing it with a built-in that's not actually a module, but hopefully still a good example of an abstracted component.

Reply:
Gotcha, sounds good